### ISSUE TITLE
[Feature Request] Configured Items added to mob drops

### ISSUE THREAD

**Issue by AzureZhen** [RobertSkalko/Mine-and-Slash#8]
Feature: Having the items you configure in the CompatibleItems config added to the mob drop list. This is something that can be added to the config like so it would still be a feature that can be turned off and on. 
```
"rcraft:dualswords": {
      "itemType": "Sword",
      "uniqueItemWeight": 0,
      "normalItemWeight": 80,
      "runedItemWeight": 20,
      "randomUniqueUpToTier": 10,
      "minRarity": 0,
      "maxRarity": 5,
      "itemIsPlayerLevel": true,
      "itemLevelIfDoesntUsePlayerLevel": 1,
      "levelVariance": 0,
      "uniqueId": "",
      "uniqueIsRandom": true,
      "useDropSystem": true
    }
```

Reasoning: I mean you already have the items and such being given stats, this is honestly the logical next step. I know you are working on the 1.13 port but would you be able to include it in the 1.12.2 version?

**Comment by RobertSkalko** [comment]
Not a bad idea, i'll have to include dropweight in there too and then another config for drop chance. But it shouldn't be hard.

Idk if i'll put it to 1.12 though but it'll surely be in 1.13 and 1.14 as long as i don't forget

**Comment by AzureZhen** [comment]
Looking to fix this issue for 1.12 release of Mine and Slash via a [Pull Request](https://github.com/RobertSkalko/Mine-and-Slash/pull/10).
